<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GastosApp</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>510</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="addGastoTab">
     <attribute name="title">
      <string>Añadir gasto</string>
     </attribute>
     <widget class="QPushButton" name="deleteCurrentBtn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>240</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 41, 41);
font: 700 10pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>ELIMINAR GASTO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 41, 41);
font: 700 10pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>BORRAR TODA LA
 TABLA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshBtn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>310</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);
font: 700 10pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>OBTENER DATOS</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>671</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="saveBtn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>100</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(115, 210, 22);
font: 700 10pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>AÑADIR GASTO</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>841</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>FECHA</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="maximum">
          <double>999999.989999999990687</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDateEdit" name="dateEdit">
         <property name="locale">
          <locale language="Spanish" country="Spain"/>
         </property>
         <property name="displayFormat">
          <string>d/M/yyyy</string>
         </property>
         <property name="timeSpec">
          <enum>Qt::UTC</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TIPO</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>COSTE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DESCRIPCIÓN</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="comboBox">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NOMBRE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_nombre"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="editBtn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>170</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 175, 62);
font: 700 10pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>EDITAR GASTO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="filterTab">
     <attribute name="title">
      <string>Filtro</string>
     </attribute>
     <widget class="QTableView" name="tableViewFilter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>601</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buscarBtn">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(115, 210, 22);
font: 700 10pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>BUSCAR</string>
      </property>
     </widget>
     <widget class="QComboBox" name="filterByCbx">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>314</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QDateEdit" name="fromDate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="locale">
          <locale language="Spanish" country="Spain"/>
         </property>
         <property name="displayFormat">
          <string>d/M/yyyy</string>
         </property>
         <property name="timeSpec">
          <enum>Qt::UTC</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>HASTA</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DESDE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDateEdit" name="toDate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="locale">
          <locale language="Spanish" country="Spain"/>
         </property>
         <property name="displayFormat">
          <string>d/M/yyyy</string>
         </property>
         <property name="timeSpec">
          <enum>Qt::UTC</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="tipoCbx">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TIPO</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="config">
     <attribute name="title">
      <string>Configuración</string>
     </attribute>
     <widget class="QTableView" name="configTable">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>351</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="editConfigBtn">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>80</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 175, 62);
font: 700 10pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>EDITAR TIPO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveTypeBtn">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(115, 210, 22);
font: 700 10pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>AÑADIR TIPO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteCurrentConfigBtn">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 41, 41);
font: 700 10pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>ELIMINAR TIPO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setDbPath">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>370</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(98, 160, 234);
font: 700 10pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>RUTA DB</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dbPathLine">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>541</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionSALIR">
   <property name="text">
    <string>SALIR</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
